### DISCUSSION TITLE
Question: How does Trivy handle vulnerabilities with aliases?

### DISCUSSION THREAD

**Discussion by BrittanyBoles7** [aquasecurity/trivy#5148] [Q&A] [answered]
### Question

Hello, I am struggling to understand how Trivy determines labels for vulnerabilities and any clarification would be appreciated.  Below the two sections are asking the same thing, but in different ways. I hope one of them makes sense. 

How are relationships between the different vulnerability databases handled? Specifically, if a vulnerability with an ID say GHSA-6q4m-7476-932w, which has an aliases vulnerability CVE-2023-27581, is found within an docker image, will Trivy also report CVE-2023-27581 as a separate vulnerability if it’s triggered by the same issue? 

Do you rename a vulnerability as its CVE equivalent if there is one? If that's the case, if the same vulnerability(same problematic lines of code etc.) also show up as a CVE ID do you report both, the finding relabeled as a CVE and the CVE?

### Target

None

### Scanner

Vulnerability

### Output Format

JSON

### Mode

None

### Operating System

ubuntu

### Version

```bash
all versions
```

**Comment by knqyf263** [comment, marked as answer]
Trivy tries to aggregate vulnerabilities by CVE-ID. GHSA-6q4m-7476-932w should be shown as an alias of CVE-2023-27581.

**Reply by BrittanyBoles7** [reply]
Thank You
